### DISCUSSION TITLE
Kaiser Index Files

### DISCUSSION THREAD

**Discussion by moodytheboss** [CMSgov/price-transparency-guide#696] [General]
Hello All,

I was trying to ingest Kaiser files. The index files of Kaiser do not include pre-defiled keys. For example, 2023-08-01_KFHP-CO_index.json includes key like PLAN_NM, which should be plan_name. 

{"PLAN_NM":"KP COLORADO OPTION SILVER X Kaiser Foundation Health Plan of Colorado","PLAN_ID_TYPE":"HIOS","PLAN_ID":"21032CO1040008","PLAN_MARKET_TYPE":"individual"}

I am not sure how does these files pass the schema validation. Just thought to share.

Regards
Akash

**Comment by nmayle** [comment]
It is strange that one year into the rule and we still see major carriers with files that are not compliant with the schema in such basic ways.

Is there any enforcement going on?
